classes/security/authorization/internal/QueryAssignedToUserAccessPolicy.inc.php

```diff
@@ -47,6 +47,12 @@ function effect() {
 		$queryDao = DAORegistry::getDAO('QueryDAO');
 		if ($queryDao->getParticipantIds($query->getId(), $user->getId())) return AUTHORIZATION_PERMIT;
 
+		// Managers are allowed to access discussions they are not participants in
+		// as long as they have Manager-level access to the workflow stage
+		$accessibleWorkflowStages = $this->getAuthorizedContextObject(ASSOC_TYPE_ACCESSIBLE_WORKFLOW_STAGES);
+		$managerAssignments = array_intersect(array(ROLE_ID_MANAGER), $accessibleWorkflowStages[$query->getStageId()]);
+		if (!empty($managerAssignments)) return AUTHORIZATION_PERMIT;
```

Comment:
@NateWr , I had a question for this part as well, but I cannot find it any more -- apparently I haven't added it at all :-P Sorry :-( I was wondering if we should maybe use here the new AssignedStageRoleHandlerOperationPolicy :-)

Reply:
Can you call another `...Policy` inside of a `...Policy::effect()`? How does that work?